test/scripts/redeploy.sh

```
@@ -48,6 +50,10 @@ DST_IMAGE="localhost/flightctl-${IMAGE}-${OS}:latest"
 podman tag "${SRC_IMAGE}" "${DST_IMAGE}"
 kind_load_image "${DST_IMAGE}"
 
+# Ensure the deployment uses the local image (fixes cases where the deployment
+# was originally created with a remote registry reference).
+${OC} set image deployment/flightctl-${IMAGE} flightctl-${IMAGE}=${DST_IMAGE} -n ${NAMESPACE} 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)